script.js

```diff
@@ -1 +1,189 @@
-//CEOS 19기 프론트엔드 파이팅🔥 ദ്ദി˶ˊᵕˋ˵)
+/*
+  checkboxNumber: todoCard가 unique한 id를 갖기 위한 변수
+*/
+let checkboxNumber = 0;
+
+//todoCard 만들기
+const createTodoCard = (todo, isDone) => {
+  const todoCard = document.createElement('div');
+  const checkbox = document.createElement('input');
+  const todoContent = document.createElement('label');
+  const todoDelete = document.createElement('div');
+
+  todoCard.className = 'todoCard';
+  checkbox.type = 'checkbox';
+  checkbox.className = 'todoCheckbox';
+  checkbox.id = `todoCheckbox${checkboxNumber}`;
+  if (isDone) {
+    checkbox.checked = true;
+    todoCard.className = 'doneCard';
+  } else {
+    todoCard.className = 'todoCard';
+  }
+  todoContent.className = 'todoContent';
+  todoContent.htmlFor = `todoCheckbox${checkboxNumber++}`;
+  todoContent.textContent = todo;
+  todoDelete.className = 'todoDelete';
+  todoDelete.textContent = 'X';
+
+  todoCard.append(checkbox, todoContent, todoDelete);
+  if (isDone) {
+    document.querySelector('.doneSection').append(todoCard);
+  } else {
+    document.querySelector('.todoSection').append(todoCard);
+  }
+};
+
+//localstorage에 저장된 todo,done 불러오기
+const todoList = JSON.parse(localStorage.getItem('todo'));
+const donelist = JSON.parse(localStorage.getItem('done'));
+if (todoList) {
+  todoList.forEach((todo) => {
+    createTodoCard(todo, false);
+  });
+}
+if (donelist) {
+  donelist.forEach((todo) => {
+    createTodoCard(todo, true);
+  });
+}
+
+//localstorage에 todo,done 추가하기
+const addTodoToLocalStorage = (todo) => {
+  const todoList = JSON.parse(localStorage.getItem('todo'));
+  localStorage.removeItem('todo');
+  if (todoList) {
+    const newTodoList = [...todoList, todo];
+    localStorage.setItem('todo', JSON.stringify(newTodoList));
+  } else {
+    localStorage.setItem('todo', JSON.stringify([todo]));
+  }
+};
+const addDoneToLocalStorage = (todo) => {
+  const doneList = JSON.parse(localStorage.getItem('done'));
+  localStorage.removeItem('done');
+  if (doneList) {
+    const newDoneList = [...doneList, todo];
+    localStorage.setItem('done', JSON.stringify(newDoneList));
+  } else {
+    localStorage.setItem('done', JSON.stringify([todo]));
+  }
+};
+
```

Comment:
addTodoToLocalStorage 함수와 addDoneToLocalStorage함수 그리고 deleteTodoFromLocalStorage와 deleteDoneFromLocalStorage 함수가 매우 유사한거 같아요. 공통 로직을 하나로 합치는 함수를 작성해보면 코드 가독성이나 유지보수에 정말 좋을 것 같습니다.
```suggestion
const updateLocalStorageList = (key, todo, isDelete = false) => {
  let list = JSON.parse(localStorage.getItem(key)) || [];
  if (isDelete) {
    list = list.filter(item => item !== todo);
  } else {
    list.push(todo);
  }
  localStorage.setItem(key, JSON.stringify(list));
};

```

예를 들면 이런식으로 함수를 수정하면 코드의 양을 줄일 수 있을 것 같아요 :)